.travis.yml

```diff
@@ -35,21 +35,10 @@ before_install:
     - set -e  # fail on any error
 
 install:
-    - source $TRAVIS_BUILD_DIR/.travis/install.sh
-    - source $TRAVIS_BUILD_DIR/.travis/install_postgres.sh
-    - source $TRAVIS_BUILD_DIR/.travis/install_pyqt.sh
-    - source $TRAVIS_BUILD_DIR/.travis/build_doc.sh
-    - pip install codecov
+    - source $TRAVIS_BUILD_DIR/.travis/stage_install.sh
```

Comment:
-1 for this change

Before, travis output grouped each install section in its own group,
now, everything is grouped together and you need to scroll to the relevant part.